suit/templatetags/suit_menu.py

```diff
@@ -16,9 +21,12 @@
 from suit.config import get_config
 
 register = template.Library()
+if django.VERSION >= (2, 0, 0, 'final', 0):
```

Comment:
`assignment_tag` has been deprecated since 1.9.

Also, you can compare against a "coarse" version, not a very specific one. Makes it a little more readable.

```python
if django.VERSION < (1, 9):
    simple_tag = register.assignment_tag
else:
    simple_tag = register.simple_tag
```